### DISCUSSION TITLE
Source in archive not found

### DISCUSSION THREAD

**Discussion by MehdiJafari84** [OvenMediaLabs/OvenMediaEngine#1179] [Help] [answered]
Hello
I'm new to OME
I want to install with source code, but in Getting Started doc the .tar.gz not found
https://airensoft.gitbook.io/ovenmediaengine/getting-started
This file not found!
https://github.com/AirenSoft/OvenMediaEngine/archive/0.15.8.tar.gz

please help me Thanks.

**Comment by getroot** [comment, marked as answer]
sorry,

https://github.com/AirenSoft/OvenMediaEngine/archive/v0.15.8.tar.gz

is the correct url. I will fix the docs

**Comment by dimiden** [comment]
@MehdiJafari84 
I'm sorry for the confusion. There was a typo while I was working on the manual.
I updated the manual, so please check it again. Thank you!

**Comment by MehdiJafari84** [comment]
Thank you for your great work
